account_bank_statement_import_qif/wizards/account_bank_statement_import_qif.py

```diff
@@ -82,12 +86,12 @@ def _parse_file(self, data_file):
 
     def _complete_stmts_vals(self, stmt_vals, journal_id, account_number):
         """Match partner_id if hasn't been deducted yet."""
-        res = super(AccountBankStatementImport, self)._complete_stmts_vals(
-            stmt_vals, journal_id, account_number,
-        )
+        res = super()._complete_stmts_vals(stmt_vals, journal_id, account_number)
         # Since QIF doesn't provide account numbers (normal behaviour is to
         # provide 'account_number', which the generic module uses to find
         # the partner), we have to find res.partner through the name
+        if not self.is_qif:
```

Comment:
Please don't create a new field. Use existing method `_check_qif`. Anyway, is this really needed?

Reply:
This is necessary so that other modules do not execute that code that is only needed in this module. 

Perhaps another possible approach would be to inject it in context here https://github.com/odoo/odoo/blob/13.0/addons/account_bank_statement_import/account_bank_statement_import.py#L50 but I didn't want to do that in order not to repeat code. 
What do you think about this?

Reply:
OK, but it should be a method, not a field.